src/map/utils/battleutils.cpp

```diff
@@ -3089,6 +3100,16 @@ namespace battleutils
             * (100 + PAttacker->getMod(Mod::SKILLCHAINBONUS)) / 100
             * (100 + PAttacker->getMod(Mod::SKILLCHAINDMG)) / 100);
 
+        auto PChar = dynamic_cast<CCharEntity *>(PAttacker);
+        if (PChar and PChar->GetMJob() == JOB_NIN and PChar->GetMLevel() >= 75)
+        {
+            auto angle = PDefender->loc.p.rotation - getangle(PDefender->loc.p, PChar->loc.p);
```

Comment:
Need to check if Yonin is active.

Reply:
Er Innin*

Reply:
Er Innin*

Reply:
Done in new push,and checked for innin :P